### DISCUSSION TITLE
Хочу помочь с продвижением игры

### DISCUSSION THREAD

**Discussion by MrTip8** [Sokol-Games-GmbH/PixelCombats.PublicApi#159] [Ideas]
Приветствую Константин! Меня зовут Mr.Tip, на данный момент, после того как волор ушел, я остался единственным достойным и популярный ютбуером рс2... Пользуюсь большим авторитетом среди комьюнити... Я бы хотел с вами поговорить в звонке(как по мне так будет куда ефективнее, чем перепиской). В звонке я бы хотел обсудить в первую очередь то, что очень легко добавить и то что будет действовительно нужно. Например новая система тетсеров, промокоды и как они могут работать в нашей игре, работа с комьюнити, поддержка киберспорта, на данный момент 1/4 всего онлайна игры это как раз таки киберы, они самые активные пользователи нашей игры, каждый день они тратят по пару часов минимум на игру. Я считаю что нам стоит поддержать их, а как хотел бы обсудить в звонке. Ещё хотел бы затронуть будущее обновления, я вижу уже много проблем от тех же скинов, хотел бы проговорить об этом с вами... У меня большой игровой опыт и не только в рс2, а и в других похожих играх, я верю что я смогу тебе помочь Костя, но для этого нам нужно связаться по звонку(в идеале), я не займу много времени, максимум 30 минут... Никаких приколов, только по делу. 
МОИ КОНТАКТЫ: 
Дискорд: Mr.Tip#8124
Телеграм: @mrtip_official
ВК: Мотора Тимофей, я вам в вк не раз писал и недавно Джаст пересылал мои сообщения... Там Джаст немного не правильно объяснил мои намерения, поэтому и пишу сюда.
НАДЕЮСЬ У НАС ПОЛУЧИТСЯ ПОГОВОРИТЬ И МЫ СМОЖЕМ СДЕЛАТЬ ИГРУ ЛУЧШЕ! 
НАПИШИТЕ МНЕ ПОЖАЛУЙСТА В ОДНУ ИЗ СОЦСЕТЕЙ ВРЕМЯ И ЧИСЛО КОГДА У ВАС ПОЛУЧИТСЯ СОЗВОНИТСЯ, СПАСИБО! УДАЧИ!

**Comment by kkohno** [comment]
насчет созвона подумаем. идеи, чтобы не потерялись, лучше тут описать все

**Reply by MrTip8** [reply]
Костя, я уже перечислил темы и считаю что в звонке будет куда ефективнее. Давай договоримся что если не хочешь там или есть другие причины, тогда так и напиши, тогда будем месяцами обсуждать все моменты в игре. Если сможешь созвониться то просто напиши ты в мои соцсети или скажи что бы я тебе куда то написал что бы договориться об созвоне. Спасибо.
